src/App.css

```diff
@@ -0,0 +1,49 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
+
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body, input, textarea, select, button {
+    font-family: 'Poppins', sans-serif;
+}
+
+.fruit-counters {
+    width: 500px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 1em;
+    margin: 0 2em;
+}
+
+.fruit-counters > * {
+    width: 500px;
+}
+
+form {
+    display: flex;
+    flex-direction: column;
+    gap: 1em;
+    width: 500px;
```

Comment:
Kijk hier hebben we dus al voor de 3e keer die width declaratie (you get the point 👍 )